### PR TITLE
typo fixes to networking

### PR DESCRIPTION
**Pull request by sascala**

### CODE REVIEW

**Comment by sascala** [comment]
@brndnmtthws - would you mind taking a quick look at this? I meant to just fix some typos, but ended up making some additions to `virtual-ip-addresses.md`.

File: 1.8/usage/service-discovery/load-balancing-vips/virtual-ip-addresses.md [hunk 1]
```diff
@@ -2,31 +2,86 @@
...
 A named VIP contains 3 components:
 
  * Private virtual IP address
  * Port (a port which the service is available on)
  * Service name
 
-You can assign a VIP to your application from the DC/OS web interface. The values you enter when you deploy a new service are translated into the appropriate `portMapping` entry in your Marathon application definition. Toggle to `JSON mode` as you create your app to see and edit your application definition.
+You can assign a VIP to your application from the DC/OS web interface. The values you enter when you deploy a new service are translated into the appropriate `portDefininitions` or `portMappings` entry in your Marathon application definition.* VIPs follow the naming convention `<service-name>.marathon.l4lb.thisdcos.directory:<port>`.
 
 ## Prerequisite:
```
Comment (sascala) [inline 1.8/usage/service-discovery/load-balancing-vips/virtual-ip-addresses.md]: Is this prerequisite really necessary?

Comment (brndnmtthws) [inline 1.8/usage/service-discovery/load-balancing-vips/virtual-ip-addresses.md]: No, I think we can drop it.

**Comment by emanic** [comment]
@sascala Some conflicts, could you resolve them? Thanks

**Comment by sascala** [comment]
OK, conflicts fixed, @emanic

**Pull request merged**